src/Bicep.LangServer/Deploy/DeploymentParametersHelper.cs

```diff
@@ -0,0 +1,181 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using Bicep.LanguageServer.Configuration;
+using Bicep.LanguageServer.Handlers;
+using Newtonsoft.Json.Linq;
+
+namespace Bicep.LanguageServer.Deploy
+{
+    public class DeploymentParametersHelper
+    {
+        // Per documentation here- https://docs.microsoft.com/en-us/azure/azure-resource-manager/bicep/parameter-files
```

Comment:
Bug on azure sdk side to track this issue:
https://github.com/Azure/azure-sdk-for-net/issues/29090

Reply:
This is not a bug. The SDK should not handle parameters file. The SDK expected parameters to be name-value pairs because that's part of the [Deployment model](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/resources/resource-manager/Microsoft.Resources/stable/2021-04-01/resources.json#L4570) swagger definition, and it's used by the ARM back-end. I've added comments to the issue.

Reply:
Ah that's interesting. Thanks for the explanation @shenglol

Reply:
Agreed.  Also, Uche agreed with me that we shouldn't support the property-bag-only version from deploy (can be removed later if it's currently implemented that way).